chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -328,17 +454,29 @@ def test_elevatePrivilege_multiple_privileges_success(self):
 
     def test_symbolic_link(self):
         """
-        Support on all Unix and Vista and above.
-
-        Not supported on Windows without elevated permissions.
+        Support on Vista and above.
         """
         symbolic_link = self.capabilities.symbolic_link
 
-        if self.os_family == 'posix':
-            self.assertTrue(symbolic_link)
-            return
+        self.assertTrue(symbolic_link)
 
-        if self.runningAsAdministrator():
-            self.assertTrue(symbolic_link)
-        else:
-            self.assertFalse(symbolic_link)
+    def test_create_home_folder(self):
+        """
+        On Windows home folders can be created if required privileges
```

Comment:
instead of "required privileges" can we be more explicit and add details about which exact privileges are requried... like Run As Admin, SE_BACKUP...etc